apps/site-projects/src/components/modules/Projects/Project/Project.js

```diff
@@ -56,7 +58,7 @@ const Project = ({ project, theme }) => {
         description={project?.description}
         images={project?.Images}
       />
-      <Role ref={roleRef} data={project?.opportunities} projectSlug={project.slug} />
+			{/*}<Role ref={roleRef} data={project?.opportunities} projectSlug={project.slug} />{*/}
```

Comment:
not sure if this is something you may want back or if it can just be removed?

Reply:
This is something that might be coming back, so I kept it in here for now for ease of access